js/pages/healthCareProvider.js

```diff
@@ -72,116 +72,116 @@ export const healthCareProvider = (siteId) => {
     <div class="col eligibility-form">
         <form method="POST" id="eligibilityForm">
             <div class="form-group">
-                <label for="827220437"><strong>Who is your healthcare provider?<span class="required hovertext" data-hover-text="Healthcare provider is required"> *</span></strong>
+                <label for="827220437"><strong data-i18n="provider.whoProvider">Who is your healthcare provider?<span class="required hovertext" data-hover-text="Healthcare provider is required"> *</span></strong>
                     <select ${siteId ? `disabled` : ``} class="form-control" style="margin-left:0px;" id="827220437" required>
-                        <option value="">-- Select healthcare provider --</option>
-    `;
+                        <option value="" data-i18n="provider.selectProvider">-- Select healthcare provider --</option>
+    `);
 
     for (let siteCode in sites()){
         template += `<option ${siteId && siteId === parseInt(siteCode) ? `selected`: `` } value=${siteCode}>${sites()[siteCode]}</option>`;
     }
 
-    template += `
+    template += translateHTML(`
                     </select>
                 </label>
             </div>
             <div class="row">
-            <span class="required">*Required</span>
+            <span class="required" data-i18n="provider.required">*Required</span>
                 <div class="ml-auto">
-                <button style="display:none;" id="openModal" class="btn btn-primary save-data consentNextButton" data-toggle="modal" data-target="#HealthProviderModal">Submit</button>
```

Comment:
I'm a little unclear on the purpose of these double Submit buttons. I do see that this is in the source file, but is this correct?